lib/ts/recipe/thirdparty/providers/types.ts

```diff
@@ -33,6 +34,12 @@ export type BuiltInProviderConfig = {
     buttonComponent?: JSX.Element;
 
```

Comment:
no tests for this change?

Reply:
We could add one, but we need to update the auth0 config to allow the test redirection url

Reply:
Let's discuss on call